sync-core/src/main/java/com/cloudant/http/HttpConnectionRequestFilter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2015 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.cloudant.http;
+
+/**
+ * Created by tomblench on 30/03/15.
+ */
+
+/**
+ A <code>HttpConnectionFilter</code> can either be configured as a Request Filter or a Response Filter:
```

Comment:
`HttpConnectionFilter` isn't a noun in our system right now, so I'm not sure it should be here -- perhaps this whole sentence should be removed.